web-client/public/js/graph.js

```diff
@@ -370,18 +370,18 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
     var weight = boundingBoxContainer.selectAll(".weight");
 
     simulation
-        .nodes(nodes)
+        .nodes(network.genes)
```

Comment:
Go over this naming with @kdahlquist —I think we intentionally chose the name “nodes” over “genes” initially because there’s nothing specific about our adjacency matrices that says that the row/column headers represent genes at all. To preserve that generality, it may be better to stick with `network.nodes`. But then, perhaps we are now moving into territory where the generalization is getting forced...?

Either way, this is worth a little discussion I think.

Reply:
The network object that the spreadsheet controller creates has always contained the property `genes`. We've just been "renaming" it to nodes in upload.js before passing it in to the `drawGraph` function. @kdahlquist @dondi should I refactor it to `nodes` in the network object itself?

Reply:
Ah OK I get it. True that an incoming GRNmap workbook _is_ gene-specific, so that is understandable. It is _GRNsight_ that is a little more generalized, and that explains the GRNsight code’s use of `nodes` once the import is finished.

This works for me then.